### DISCUSSION TITLE
Platform "android-arm64" not supported by class

### DISCUSSION THREAD

**Discussion by jianzhangru** [bytedeco/javacpp-presets#1180] [Q&A] [answered]
Hi, I got the following error when I ran the example code on Android:

java.lang.UnsatisfiedLinkError: Platform "android-arm64" not supported by class org.bytedeco.tensorflowlite.FlatBufferModel

In my gradle file, I've added:
implementation group: 'org.bytedeco', name: 'tensorflow-lite-platform', version: '2.8.0-1.5.7'

And it failed on this line:
FlatBufferModel model = FlatBufferModel.BuildFromFile(modelPath)

Any thoughts? Thank you.

**Comment by saudet** [comment, marked as answer]
There wasn't any builds for Android in that release, but since TF Lite was designed for platforms like Android, it's not hard to create builds for it, so I've done just that in commit https://github.com/bytedeco/javacpp-presets/commit/b0f62702f21d32d3b8c980dd551c1c0347ce443a. Please give it a try with the snapshots: http://bytedeco.org/builds/

**Reply by saudet** [reply]
And they have now been released with version 1.5.8. Enjoy!
